### PR TITLE
Fixing version string to match official binary version string

### PR DESCRIPTION
**Pull request by arjunurs**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
Fixing version string to match official binary version string

### CODE REVIEW

File: Formula/terragrunt.rb [hunk 1]
```diff
@@ -22,7 +22,7 @@ def install
     mkdir_p buildpath/"src/github.com/gruntwork-io/"
     ln_s buildpath, buildpath/"src/github.com/gruntwork-io/terragrunt"
     system "glide", "install"
-    system "go", "build", "-o", bin/"terragrunt", "-ldflags", "-X main.VERSION=" + version.to_s
+    system "go", "build", "-o", bin/"terragrunt", "-ldflags", "-X main.VERSION=v" + version.to_s
```
Comment (MikeMcQuaid) [inline Formula/terragrunt.rb]: May as well do `"-X main.VERSION=v#{version}"` while you're here 👍

Comment (arjunurs) [inline Formula/terragrunt.rb]: @MikeMcQuaid Updated according to your recommendation.

**Comment by arjunurs** [comment]
@MikeMcQuaid Updated according to your recommendation.

**Comment by MikeMcQuaid** [comment]
Thanks again for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format is `terragrunt: match version with official binary.`. Please complete the issue template in future PRs where this format is detailed in the linked [Contributing guidelines](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md). Thanks!

**Pull request closed without merging**